packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -944,10 +979,15 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
 
     return itemToString(value)
   }
+
+  private isValueEmpty = (value: ShorthandValue | ShorthandValue[]) => {
```

Comment:
`ShorthandValue` should be enough for the type here since it's already covering the array type
![screenshot 2019-02-12 at 14 31 33](https://user-images.githubusercontent.com/5442794/52638760-f0d4d280-2ed2-11e9-9755-b91c3cf878b1.png)

Reply:
Actually we have a dedicated type for this `ShorthandCollection`, replaced `ShorthandValue[]` with it